functions/function/header-enricher-function/src/test/java/org/springframework/cloud/fn/header/enricher/HeaderEnricherFunctionApplicationTests.java

```diff
@@ -42,7 +41,6 @@
 		"header.enricher.headers=foo='bar' \\n baz='fiz' \\n buz=payload \\n jaz=@value",
 		"header.enricher.overwrite = true" })
 @DirtiesContext
-@Disabled
```

Comment:
Can you also re-enable 
`org.springframework.cloud.stream.app.processor.header.enricher.HeaderEnricherProcessorTests`? I missed this one in the issue description. I can also do it after this is merged if you are busy. Thanks for the quick fix @artembilan . I'm glad it was straightforward.